python/ray/_private/runtime_env/uv.py

```diff
@@ -117,11 +119,42 @@ async def _check_uv_existence(
 
         try:
             # If `uv` doesn't exist, exception will be thrown.
-            await check_output_cmd(check_existence_cmd, logger=logger, cwd=cwd, env=env)
-            return True
+            version_output = await check_output_cmd(
+                check_existence_cmd, logger=logger, cwd=cwd, env=env
+            )
+
+            # If exists, the output format would look like
+            # uv <version> (<sha> <release date>), for example,
+            # uv 0.5.1 (f399a5271 2024-11-08)
+            version_strs = version_output.split()
+            if len(version_strs) == 4 and version_strs[0] == "uv":
+                return version_strs[1]
```

Comment:
I think this is fragile. The output format can change at anytime. Do we have other way to get the version?